support/prepReleaseCommit.ts

```diff
@@ -82,7 +81,7 @@ import yargs from "yargs";
     // we keep track of `beta` and `next` releases since `standard-version` resets the version number when going in between
     // this should not be needed after v1.0.0 since there would no longer be a beta version to keep track of
     const targetDescendingOrderTags = semverTags.filter((tag) => targetVersionPattern.test(tag)).sort(semver.rcompare);
-    const targetReleaseVersion = semver.inc(targetDescendingOrderTags[0], "prerelease", target);
+    const targetReleaseVersion = semver.inc(targetDescendingOrderTags[0], "prerelease", target) || "";
```

Comment:
Sidebar: maybe we should throw here instead of falling back to `""` to make it clear that the target version is invalid.